src/components/oval-mode/oval.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="20px" height="20px" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 43.2 (39069) - http://www.bohemiancoding.com/sketch -->
+    <title>oval</title>
+    <desc>Created with Sketch.</desc>
```

Comment:
These SVG descs aren't surfaced somehow in the UI are they? It would be weird if you were using a screen reader and it kept saying "Created with Sketch"

Reply:
I don't think so, because I think they end up being encoded as data uri images.